### PR TITLE
doc: Update the ownership of RST files

### PR DESCRIPTION
**Pull request by umapraseeda**
Add the ownership of all the RST files and txt files to the relevant techwriter groups assigned to the different software teams

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -55,14 +68,208 @@
 /cmake/sysbuild/suit_utilities.cmake      @nrfconnect/ncs-charon
 
 # All doc related files
-/doc/					  @nrfconnect/ncs-co-doc
+/doc/                                     @nrfconnect/ncs-co-doc
```
Comment (carlescufi) [inline CODEOWNERS:71]: Please remove this one, we need to ensure that we catch missing entries and this will mask them

File: CODEOWNERS [hunk 2]
```diff
@@ -55,14 +68,208 @@
...
 /doc/_zoomin/                             @nrfconnect/ncs-co-doc @nrfconnect/ncs-doc-leads
-/doc/**/*.rst				  @nrfconnect/ncs-doc-owners
-/doc/**/*.svg				  @nrfconnect/ncs-doc-owners
-/doc/**/*.png				  @nrfconnect/ncs-doc-owners
-/doc/nrf/protocols/thread/*.rst           @wiba-nordic
-/doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ @FrancescoSer
-/doc/nrf/app_dev/device_guides/nrf54l/    @annwoj
+/doc/kconfig/*.rst                        @nrfconnect/ncs-doc-leads
```
Comment (carlescufi) [inline CODEOWNERS:73]: Inside the `/doc/` folder you do not need the `*.rst` endings, just use:

`/doc/kconfig/`

and so on

**Review by carlescufi — CHANGES_REQUESTED** [review]

**Comment by carlescufi** [comment]
@umapraseeda  can you take this out of draft please?

File: CODEOWNERS [hunk 3]
```diff
@@ -55,14 +67,208 @@
...
-/doc/**/*.rst				  @nrfconnect/ncs-doc-owners
-/doc/**/*.svg				  @nrfconnect/ncs-doc-owners
-/doc/**/*.png				  @nrfconnect/ncs-doc-owners
-/doc/nrf/protocols/thread/*.rst           @wiba-nordic
-/doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ @FrancescoSer
+/doc/kconfig/                             @nrfconnect/ncs-doc-leads
+/doc/matter/                              @nrfconnect/ncs-matter-doc
+/doc/mcuboot/                             @nrfconnect/ncs-pluto-doc
+/doc/nrf/                                 @nrfconnect/ncs-doc-leads
+/doc/nrf/app_dev/**/                      @nrfconnect/ncs-doc-leads @nrfconnect/ncs-vestavind-doc
```
Comment (carlescufi) [inline CODEOWNERS:75]:
```suggestion
/doc/nrf/app_dev/                      @nrfconnect/ncs-doc-leads @nrfconnect/ncs-vestavind-doc
```

File: CODEOWNERS [hunk 4]
```diff
@@ -55,14 +67,208 @@
...
 /doc/_zoomin/                             @nrfconnect/ncs-co-doc @nrfconnect/ncs-doc-leads
-/doc/**/*.rst				  @nrfconnect/ncs-doc-owners
-/doc/**/*.svg				  @nrfconnect/ncs-doc-owners
-/doc/**/*.png				  @nrfconnect/ncs-doc-owners
-/doc/nrf/protocols/thread/*.rst           @wiba-nordic
-/doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ @FrancescoSer
+/doc/kconfig/                             @nrfconnect/ncs-doc-leads
+/doc/matter/                              @nrfconnect/ncs-matter-doc
+/doc/mcuboot/                             @nrfconnect/ncs-pluto-doc
+/doc/nrf/                                 @nrfconnect/ncs-doc-leads
```
Comment (carlescufi) [inline CODEOWNERS:74]:
```suggestion
/doc/nrf/*.rst                                @nrfconnect/ncs-doc-leads
```

File: CODEOWNERS [hunk 5]
```diff
@@ -55,14 +67,208 @@
...
+/doc/nrf/app_dev/**/                      @nrfconnect/ncs-doc-leads @nrfconnect/ncs-vestavind-doc
+/doc/nrf/app_dev/bootloaders_dfu/**/      @nrfconnect/ncs-pluto-doc
```
Comment (carlescufi) [inline CODEOWNERS:76]:
```suggestion
/doc/nrf/app_dev/bootloaders_dfu/      @nrfconnect/ncs-pluto-doc
```

File: CODEOWNERS [hunk 6]
```diff
@@ -55,14 +67,208 @@
...
 /doc/nrf/app_dev/device_guides/nrf54l/    @annwoj
+/doc/nrf/app_dev/device_guides/nrf70/     @nrfconnect/ncs-wifi-doc
+/doc/nrf/app_dev/device_guides/nrf91/     @nrfconnect/ncs-modem-doc
+/doc/nrf/app_dev/device_guides/nrf91/nrf91_snippet.rst @nrfconnect/ncs-cia-doc
+/doc/nrf/app_dev/device_guides/pmic/      @nrfconnect/ncs-pmic-doc
+/doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ @FrancescoSer
+/doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_*.rst @nrfconnect/ncs-charon-doc
+/doc/nrf/app_dev/device_guides/nrf54h.rst @FrancescoSer
+/doc/nrf/app_dev/device_guides/wifi_coex.rst @nrfconnect/ncs-doc-leads
+/doc/nrf/dev_model_and_contributions/**/  @nrfconnect/ncs-doc-leads @nrfconnect/ncs-vestavind-doc
```
Comment (carlescufi) [inline CODEOWNERS:90]:
```suggestion
/doc/nrf/dev_model_and_contributions/  @nrfconnect/ncs-doc-leads @nrfconnect/ncs-vestavind-doc
```

File: CODEOWNERS [hunk 7]
```diff
@@ -55,14 +67,208 @@
...
+/doc/nrf/gsg_guides/                      @nrfconnect/ncs-doc-leads @nrfconnect/ncs-vestavind-doc @nrfconnect/ncs-wayland-doc
+/doc/nrf/gsg_guides/thingy91_gsg.rst      @nrfconnect/ncs-cia-doc
+/doc/nrf/gsg_guides/nrf9160_gs.rst        @nrfconnect/ncs-modem-doc
+/doc/nrf/gsg_guides/nrf7002_gs.rst        @nrfconnect/ncs-wifi-doc
+/doc/nrf/includes/*.txt                   @nrfconnect/ncs-doc-leads
+/doc/nrf/includes/boardname_tables/sample_boardnames.txt @nrfconnect/ncs-co-doc
+/doc/nrf/installation/                    @nrfconnect/ncs-doc-leads @nrfconnect/ncs-vestavind-doc @nrfconnect/ncs-wayland-doc
+/doc/nrf/libraries/bin/                   @nrfconnect/ncs-doc-leads
+/doc/nrf/libraries/bin/lwm2m_carrier/     @nrfconnect/ncs-carrier-doc
+/doc/nrf/libraries/bluetooth_services/**/ @nrfconnect/ncs-si-muffin-doc @nrfconnect/ncs-dragoon-doc
```
Comment (carlescufi) [inline CODEOWNERS:118]:
```suggestion
/doc/nrf/libraries/bluetooth_services/ @nrfconnect/ncs-si-muffin-doc @nrfconnect/ncs-dragoon-doc
```

File: CODEOWNERS [hunk 8]
```diff
@@ -55,14 +67,208 @@
...
+/doc/nrf/libraries/bluetooth_services/mesh/**/ @nrfconnect/ncs-paladin-doc
```
Comment (carlescufi) [inline CODEOWNERS:119]:
```suggestion
/doc/nrf/libraries/bluetooth_services/mesh/ @nrfconnect/ncs-paladin-doc
```

File: CODEOWNERS [hunk 9]
```diff
@@ -55,14 +67,208 @@
...
+/doc/nrf/external_comp/coremark.rst       @nrfconnect/ncs-si-bluebagel-doc
+/doc/nrf/external_comp/dult.rst           @nrfconnect/ncs-si-bluebagel-doc
+/doc/nrf/external_comp/edge_impulse.rst   @nrfconnect/ncs-si-muffin-doc
+/doc/nrf/external_comp/memfault.rst       @nrfconnect/ncs-cia-doc
+/doc/nrf/external_comp/nrf_cloud.rst      @nrfconnect/ncs-nrf-cloud-doc
+/doc/nrf/gsg_guides/                      @nrfconnect/ncs-doc-leads @nrfconnect/ncs-vestavind-doc @nrfconnect/ncs-wayland-doc
+/doc/nrf/gsg_guides/thingy91_gsg.rst      @nrfconnect/ncs-cia-doc
+/doc/nrf/gsg_guides/nrf9160_gs.rst        @nrfconnect/ncs-modem-doc
+/doc/nrf/gsg_guides/nrf7002_gs.rst        @nrfconnect/ncs-wifi-doc
+/doc/nrf/includes/*.txt                   @nrfconnect/ncs-doc-leads
```
Comment (carlescufi) [inline CODEOWNERS:113]:
```suggestion
/doc/nrf/includes/                   @nrfconnect/ncs-doc-leads
```

File: CODEOWNERS [hunk 10]
```diff
@@ -55,14 +67,208 @@
...
+/doc/nrf/libraries/bluetooth_services/mesh.rst @nrfconnect/ncs-paladin-doc
+/doc/nrf/libraries/bluetooth_services/services/fast_pair.rst @nrfconnect/ncs-si-bluebagel-doc
+/doc/nrf/libraries/bluetooth_services/services/wifi_prov.rst @nrfconnect/ncs-wifi-doc
+/doc/nrf/libraries/caf/                   @nrfconnect/ncs-si-muffin-doc @nrfconnect/ncs-si-bluebagel-doc
+/doc/nrf/libraries/debug/cpu_load.rst     @nrfconnect/ncs-doc-leads
+/doc/nrf/libraries/debug/etb_trace.rst    @nrfconnect/ncs-cia-doc
+/doc/nrf/libraries/debug/index.rst        @nrfconnect/ncs-doc-leads
+/doc/nrf/libraries/debug/memfault_ncs.rst @nrfconnect/ncs-cia-doc
+/doc/nrf/libraries/debug/ppi_trace.rst    @nrfconnect/ncs-doc-leads
+/doc/nrf/libraries/dfu/**/                @nrfconnect/ncs-pluto-doc
```
Comment (carlescufi) [inline CODEOWNERS:130]:
```suggestion
/doc/nrf/libraries/dfu/                @nrfconnect/ncs-pluto-doc
```

File: CODEOWNERS [hunk 11]
```diff
@@ -55,14 +67,208 @@
...
+/doc/nrf/libraries/networking/mqtt_helper.rst @nrfconnect/ncs-cia-doc
+/doc/nrf/libraries/networking/nrf_cloud.rst @nrfconnect/ncs-nrf-cloud-doc
+/doc/nrf/libraries/networking/nrf_cloud_*.rst @nrfconnect/ncs-nrf-cloud-doc
+/doc/nrf/libraries/networking/nrf_provisioning.rst @nrfconnect/ncs-iot-oulu-tampere-doc
+/doc/nrf/libraries/networking/rest_client.rst @nrfconnect/ncs-iot-positioning-doc
+/doc/nrf/libraries/networking/softap_wifi_provision.rst @nrfconnect/ncs-cia-doc
+/doc/nrf/libraries/networking/wifi_credentials.rst @nrfconnect/ncs-cia-doc
+/doc/nrf/libraries/networking/wifi_mgmt_ext.rst @nrfconnect/ncs-cia-doc
+/doc/nrf/libraries/networking/wifi_ready.rst @nrfconnect/ncs-wifi-doc
+/doc/nrf/libraries/nfc/**/                @nrfconnect/ncs-si-muffin-doc
```
Comment (carlescufi) [inline CODEOWNERS:175]:
```suggestion
/doc/nrf/libraries/nfc/                @nrfconnect/ncs-si-muffin-doc
```

File: CODEOWNERS [hunk 12]
```diff
@@ -55,14 +67,208 @@
...
+/doc/nrf/libraries/security/fatal_error.rst @nrfconnect/ncs-doc-leads
+/doc/nrf/libraries/security/hw_unique_key.rst @nrfconnect/ncs-aegir-doc
+/doc/nrf/libraries/security/identity_key.rst @nrfconnect/ncs-aegir-doc
+/doc/nrf/libraries/security/index.rst     @nrfconnect/ncs-aegir-doc
+/doc/nrf/libraries/security/trusted_storage.rst @nrfconnect/ncs-aegir-doc
+/doc/nrf/libraries/shell/                 @nrfconnect/ncs-doc-leads
+/doc/nrf/libraries/zigbee/                @nrfconnect/ncs-terahertz-doc
+/doc/nrf/libraries/index.rst              @nrfconnect/ncs-doc-leads
+/doc/nrf/protocols/amazon_sidewalk/       @nrfconnect/ncs-sidewalk-doc
+/doc/nrf/protocols/bt/bt_mesh/**/         @nrfconnect/ncs-paladin-doc
```
Comment (carlescufi) [inline CODEOWNERS:219]:
```suggestion
/doc/nrf/protocols/bt/bt_mesh/         @nrfconnect/ncs-paladin-doc
```

File: CODEOWNERS [hunk 13]
```diff
@@ -55,14 +67,208 @@
...
+/doc/nrf/protocols/bt/bt_qualification/   @nrfconnect/ncs-dragoon-doc
+/doc/nrf/protocols/bt/                    @nrfconnect/ncs-dragoon-doc
+/doc/nrf/protocols/coexistence/           @nrfconnect/ncs-doc-leads
+/doc/nrf/protocols/dect/                  @nrfconnect/ncs-modem-doc
+/doc/nrf/protocols/esb/                   @nrfconnect/ncs-si-muffin-doc
+/doc/nrf/protocols/gazell/                @nrfconnect/ncs-si-muffin-doc
+/doc/nrf/protocols/lte/                   @nrfconnect/ncs-carrier-doc
+/doc/nrf/protocols/matter/**/             @nrfconnect/ncs-matter-doc
```
Comment (carlescufi) [inline CODEOWNERS:227]:
```suggestion
/doc/nrf/protocols/matter/             @nrfconnect/ncs-matter-doc
```

File: CODEOWNERS [hunk 14]
```diff
@@ -55,14 +67,208 @@
...
+/doc/nrf/protocols/multiprotocol/         @nrfconnect/ncs-doc-leads
+/doc/nrf/protocols/nfc/                   @nrfconnect/ncs-si-muffin-doc
+/doc/nrf/protocols/thread/**/             @nrfconnect/ncs-terahertz-doc
```
Comment (carlescufi) [inline CODEOWNERS:230]:
```suggestion
/doc/nrf/protocols/thread/             @nrfconnect/ncs-terahertz-doc
```

File: CODEOWNERS [hunk 15]
```diff
@@ -55,14 +67,208 @@
...
+/doc/nrf/protocols/wifi/**/               @nrfconnect/ncs-wifi-doc
```
Comment (carlescufi) [inline CODEOWNERS:231]:
```suggestion
/doc/nrf/protocols/wifi/               @nrfconnect/ncs-wifi-doc
```

File: CODEOWNERS [hunk 16]
```diff
@@ -55,14 +67,208 @@
...
+/doc/nrf/protocols/zigbee/                @nrfconnect/ncs-terahertz-doc
+/doc/nrf/releases_and_maturity/**/        @nrfconnect/ncs-doc-release
```
Comment (carlescufi) [inline CODEOWNERS:233]:
```suggestion
/doc/nrf/releases_and_maturity/        @nrfconnect/ncs-doc-release
```

File: CODEOWNERS [hunk 17]
```diff
@@ -55,14 +67,208 @@
...
+/doc/nrf/samples/suit.rst                 @nrfconnect/ncs-charon-doc
+/doc/nrf/samples/tfm.rst                  @nrfconnect/ncs-aegir-doc
+/doc/nrf/samples/thread.rst               @nrfconnect/ncs-terahertz-doc
+/doc/nrf/samples/wifi.rst                 @nrfconnect/ncs-wifi-doc
+/doc/nrf/samples/wifi_provisioning.rst    @nrfconnect/ncs-wifi-doc
+/doc/nrf/samples/wifi_zephyr.rst          @nrfconnect/ncs-wifi-doc
+/doc/nrf/samples/zigbee.rst               @nrfconnect/ncs-terahertz-doc
+/doc/nrf/security/                        @nrfconnect/ncs-aegir-doc
+/doc/nrf/templates/                       @nrfconnect/ncs-doc-leads
+/doc/nrf/test_and_optimize/**/            @nrfconnect/ncs-doc-leads
```
Comment (carlescufi) [inline CODEOWNERS:266]:
```suggestion
/doc/nrf/test_and_optimize/            @nrfconnect/ncs-doc-leads
```

**Review by carlescufi — CHANGES_REQUESTED** [review]

**Review by gmarull — APPROVED** [review]

**Review by b-gent — APPROVED** [review]

**Pull request merged**